config/testdata/helmchart-from-oci/notation.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: source.toolkit.fluxcd.io/v1beta2
+apiVersion: source.toolkit.fluxcd.io/v1
```

Comment:
what about the other files in `config/testdata/git` and `config/testdata/helmchart-valuesfile`?

Reply:
I would leave all of these of on the beta version to ensure backwards comparability. I only bumped the samples who are part of e2e.